### PR TITLE
Remove legacy code leftover from ATI

### PR DESCRIPTION
**Pull request by achow101**
Followup to #4. Basically deletes everything that was commented out in #4 and then deletes the dead code that was used by the commented out parts.

This is to the right branch.

### CODE REVIEW

File: ArmoryQt.py [hunk 1]
```diff
@@ -631,10 +619,6 @@ def msrevsign():
       self.menusList[MENUS.Wallets].addSeparator()
       self.menusList[MENUS.Wallets].addAction(actRecoverWlt)
 
-      # def execVersion():
-      #    self.explicitCheckAnnouncements()
-      #    self.mainDisplayTabs.setCurrentIndex(self.MAINTABS.Announce)
-
       execAbout   = lambda: DlgHelpAbout(self).exec_()
       # execTrouble = lambda: webbrowser.open('https://bitcoinarmory.com/troubleshooting/')
```
Comment (fanquake) [inline ArmoryQt.py]: Remove these lines as well?

**Comment by fanquake** [comment]
If your removing annoucefetch.py, should you be removing release_scripts/annoucemap.txt, dlmap.txt etc?

File: qtdefines.py [hunk 2]
```diff
@@ -38,16 +38,6 @@
 VERTICAL = 'vertical'
 HORIZONTAL = 'horizontal'
 CHANGE_ADDR_DESCR_STRING = '[[ Change received ]]'
-HTTP_VERSION_FILE = 'https://bitcoinarmory.com/versions.txt'
-BUG_REPORT_URL = 'https://bitcoinarmory.com/scripts/receive_debug.php'
-PRIVACY_URL = 'https://bitcoinarmory.com/privacy-policy'
```
Comment (fanquake) [inline qtdefines.py]: There is a usage of PRIVACY_URL on line 3978 of ArmoryQt that needs to be removed.

**Comment by goatpig** [comment]
I don't know that I want to get rid of the announcer code yet. I may want it converted to checking the github release page to warn users of new versions. This needs to be discussed more thoroughly.

**Comment by goatpig** [comment]
@fanquake 

That's sample code essentially, inactive in the code base but useful for the repo.

**Comment by achow101** [comment]
@goatpig well my thinking was that if you really need it, you can go back into the version history and copy the code from there. I don't think that having it in the active codebase is necessary.

**Comment by droark** [comment]
utACK

Code deletion looks good. Nothing jumped out at me as unusual.

Regarding keeping the announcer code in there, I'd argue that, if it is cut (I prefer leaving it in but I have no strong opinion on that end), it should be in a separate PR, or at least its own commit. One thing that drives me mad is when devs make it difficult to find deleted code. ("Code fixes" isn't a good commit name, damn it!) It'd be good to be able to go through the commit history and see "Remove announce code" or something similar.

**Comment by achow101** [comment]
Rebased onto current dev.

**Comment by goatpig** [comment]
Will need a rebase after the translation PR merge. I'd rather merge that one first.

**Comment by achow101** [comment]
Ready for merge.

**Pull request merged**